plugins/tfidf.cc

```diff
@@ -23,9 +22,73 @@
 #include "mldb/server/analytics.h"
 #include "mldb/types/any_impl.h"
 #include "mldb/types/optional_description.h"
+#include "mldb/vfs/filter_streams.h"
 
 using namespace std;
 
+namespace {
+void
+serialize(ML::DB::Store_Writer & store, 
```

Comment:
my thought was that we would have both TF-IDF and K-Means serialized to CSVs-with-headers.